source/index.html

```diff
@@ -1,12 +1,147 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class = "page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Седона</title>
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+  <body class="page__body">
+    <header class="main-header">
+      <a class="main-header__logo">
+        <img src="../images/logo_sedona.svg" width="185" height="96" alt="Седона">
+      </a>
+      <nav class="main-nav">
+        <ul class="main-nav__list site-list">
+          <li class="site-list__item site-list__item--active">
+            <a>
+              Главная
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a href="#">
+              Фото и видео
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a href="#">
+              Форма отзыва
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a href="https://htmlacademy.ru/">
+              HTML Academy
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-container">
+      <h1 class="main-container__visually-hidden">Расскажем про Седону и причины посетить этот городок!</h1>
```

Comment:
вот как раз visually-hidden может быть отдельным блоком. Ты его свойства пропишешь один раз в этом блоке, а сам класс сможешь добавлять любому элементу
```suggestion
      <h1 class="main-container__title visually-hidden">Расскажем про Седону и причины посетить этот городок!</h1>
```